test/util/malicious/out_of_order_builder.go

```diff
@@ -20,8 +22,8 @@ type ExportFn func(builder *square.Builder) (square.Square, error)
 // in the square and which have all PFBs trailing regular transactions. Note, this function does
 // not check the underlying validity of the transactions.
 // Errors should not occur and would reflect a violation in an invariant.
-func Build(txs [][]byte, appVersion uint64, maxSquareSize int, efn ExportFn) (square.Square, [][]byte, error) {
-	builder, err := square.NewBuilder(maxSquareSize, appconsts.SubtreeRootThreshold(appVersion))
+func Build(txs [][]byte, _ uint64, maxSquareSize int, efn ExportFn) (square.Square, [][]byte, error) {
```

Comment:
these `appVersion` args can likely be removed later since they will be discarded from here moving forward